index.js

```diff
@@ -1,66 +1,80 @@
-const books = [];
-let retrievedBooks;
-if (localStorage.getItem('books') === null) {
-  retrievedBooks = localStorage.setItem('books', JSON.stringify(books));
-} else {
-  retrievedBooks = JSON.parse(localStorage.getItem('books'));
+/* eslint-disable  max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
-const displayBooks = () => {
-  retrievedBooks = JSON.parse(localStorage.getItem('books'));
-  retrievedBooks.forEach((book) => {
+
+class Storage {
+  static addBook(book) {
+    const books = Storage.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books')) {
+      books = JSON.parse(localStorage.getItem('books'));
+    } else {
+      books = [];
+    }
+    return books;
+  }
+
+  static removeBook(author) {
+    const newBooks = Storage.getBooks();
+    newBooks.forEach((book, index) => {
+      if (book.author === author) {
+        newBooks.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(newBooks));
+  }
+}
```

Comment:
- [ ] There's a bug when you remove book, to see it follow this steps:

1. Create 5 books with the same author and title, I use `title: aaaaa author: aaa`
2. Delete the first one you created, the book on the top
3. Refresh the page

If you do it you could see how another book is deleted. Please fix this.
- Hint: A good way to fix it is to assign ids for each element on the list, and then delete it according to the id in place author or title. 🐶

Reply:
Thanks for the review, But why would anyone need to add the same book 5 times in a real-life situation. Those fields would be marked as unique